spaces/debaseonomics/index.json

```
@@ -14,7 +14,7 @@
 			}
 		}
 	],
-	"members": [ "0x7b87Fb3937C2725E804304f4E3528B20ec46a5c6" ],
+	"members": [ "" ],
```

Please remove also quotes.
```suggestion
	"members": [],
```